### PR TITLE
PLAT-134298: Migrate storybook to 6

### PR DESCRIPTION
**Pull request by jeongran1230**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [X] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Need to storybook migration to 6.0

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Update sampler configuration for storybook 6 migration
- add main.js, preview.js, manager.js files
- use global parameters or decorators
- set `withKnobs` options of @storybook/addon-knobs (see https://www.npmjs.com/package/@storybook/addon-knobs)
- Add `@storybook/addon-docs` which replaces the deprecated `addon-info`

Upgrade storybook stories to Component Stories Format

Fixed: `sampler` naming components

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-129635

### Comments
Enact-DCO-1.0-Signed-off-by: Jeongran Jang (jeongran.jang@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
I get below warning. Would you take a look? @jeongran1230 
```
browser.js:39 CSF .story annotations deprecated; annotate story functions directly:
- StoryFn.story.name => StoryFn.storyName
- StoryFn.story.(parameters|decorators) => StoryFn.(parameters|decorators)
See https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#hoisted-csf-annotations for details and codemod.
```

**Review by MikyungKim — COMMENTED** [review]
I'm not sure why but layout sampler works differently from the old one.
v5 looked
image
now with your change,
image

File: packages/sampler/.storybook/preview.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+addDecorator(withKnobs);
+export const parameters = {
+	knobs: {
+		timestamps: true,
+	},
+	docs: {
+		container: DocsContainer,
+		page: DocsPage,
+		iframeHeight: 300,
+		theme: themes.light
```
Comment (MikyungKim) [inline packages/sampler/.storybook/preview.js:19]: Where these settings come from?

File: packages/sampler/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
 
+### Added
+
+- `addon-docs` to substitude `addon-info`
+
+### Changed
+
+- `addon-knobs`and `addon-actions` registration
+- `sampler` configuration by updating to Storybook 6
+- `sampler` naming components
```
Comment (MikyungKim) [inline packages/sampler/CHANGELOG.md:13]:
```suggestion
## [unreleased]

### Changed

- Migrate Storybook to 6

```

File: packages/sampler/.storybook/manager.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { create} from '@storybook/theming/create';
+import { addons } from '@storybook/addons';
+import { themes } from '@storybook/theming';
```
Comment (MikyungKim) [inline packages/sampler/.storybook/manager.js:3]:
```suggestion
import {create} from '@storybook/theming/create';
import {addons} from '@storybook/addons';
import {themes} from '@storybook/theming';
```

File: packages/sampler/.storybook/preview.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import {addDecorator} from '@storybook/react';
+import {configureActions} from '@enact/storybook-utils/addons/actions';
+import {withKnobs} from '@enact/storybook-utils/addons/knobs';
+import {DocsPage, DocsContainer} from '@enact/storybook-utils/addons/docs';
+import { themes } from '@storybook/theming';
```
Comment (MikyungKim) [inline packages/sampler/.storybook/preview.js:5]:
```suggestion
import {configureActions} from '@enact/storybook-utils/addons/actions';
import {DocsPage, DocsContainer} from '@enact/storybook-utils/addons/docs';
import {withKnobs} from '@enact/storybook-utils/addons/knobs';
import {addDecorator} from '@storybook/react';
import {themes} from '@storybook/theming';
```

File: packages/sampler/.storybook/preview.js (hunk 1 above)
Comment (jeongran1230) [reply packages/sampler/.storybook/preview.js:19]: Those are found in storybook examples, 
https://github.com/storybookjs/storybook/blob/next/examples/official-storybook/preview.js#L161
https://github.com/storybookjs/storybook/blob/next/examples/vue-cli/.storybook/preview.js#L3

File: packages/sampler/stories/default/VirtualGridList.js [hunk 5]
```diff
@@ -58,31 +57,34 @@ updateDataSize(defaultDataSize);
...
+		noScrollByWheel={boolean('noScrollByWheel', VirtualGridListConfig)}
+		onScrollStart={action('onScrollStart')}
+		onScrollStop={action('onScrollStop')}
+		scrollMode={select('scrollMode', prop.scrollModeOption, VirtualGridListConfig)}
+		spacing={ri.scale(number('spacing', VirtualGridListConfig, 20))}
+		verticalScrollbar={select('verticalScrollbar', prop.scrollbarOption, VirtualGridListConfig)}
+	/>
+);
+
+VirtualListVirtualGridList.storyNname = 'VirtualList.VirtualGridList';
```
Comment (MikyungKim) [inline packages/sampler/stories/default/VirtualGridList.js:85]:
```suggestion
VirtualListVirtualGridList.storyName = 'VirtualList.VirtualGridList';
```

**Comment by MikyungKim** [comment]
Closing in favor of #2898

**Pull request closed without merging**